types/schema.d.ts

```diff
@@ -9,6 +9,10 @@ export interface FastifySchema {
   querystring?: unknown;
   params?: unknown;
   headers?: unknown;
+  response?: {
+    [code: number]: unknown;
```

Comment:
please make this `[code: number | string]: unknown`

Reply:
Just realizing that this syntax is invalid. Oops my mistake!

Reply:
I'd rather this be `unknown` like the rest of the properties. Then encourage the user to specify it themselves using overloading or generics.

Reply:
It does give more flexibility in terms of typing your schema. I'll change it. 👍